tests/test_client.py

```diff
@@ -48,7 +44,9 @@ def test_index_list_with_params(swg_index_client):
     }
     data2['urls'] = ['s3://endpointurl/bucket/key_2', 's3://anotherurl/bucket/key_2']
     data2['urls_metadata'] = {
-        's3://endpointurl/bucket/key_2': {'state': 'error', 'other': 'xxx'}}
+        's3://endpointurl/bucket/key_2': {'state': 'error', 'other': 'xxx'},
```

Comment:
This duplication is suspicious to me of urls_metadata's keys and urls. I just want to know why.

Reply:
I think you found the answer in one of the previous comments. I kept the input interface the same for backwards compatibility. That means indexclient, and the applications that use it, do not need to change and will continue to work as normal.

Changing the interface was a little out of the scope of this ticket, but it can be changed in the future much easier after this PR.